contrib/helm/Makefile

```diff
@@ -0,0 +1,44 @@
+# create kind cluster with nginx-ingress as the most popular ingress controller for K8S
+.PHONY: deploy
+create-cluster:
+	kind create cluster --name oauth2-proxy --config kind-cluster.yaml
+	make setup-dns
+	make setup-ingress
+
+.PHONY: setup-ingress
+setup-ingress:
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+	sleep 5 # because of https://github.com/kubernetes/kubernetes/issues/83242
+	kubectl --namespace ingress-nginx wait --for=condition=available --timeout=30s deployment/ingress-nginx-controller
+	kubectl --namespace ingress-nginx rollout status --timeout 90s deployment/ingress-nginx-controller
```

Comment:
When I tried this earlier, 90s was too short and it errored out for me. Is there any harm upping this to say 5m which would be inline with the help deployments?

Reply:
sounds good, will do. One of the thing I prefer helm is that it has a built-in readiness check whereas with kubectl I had to insert those checks.

Reply:
What's the rationale for applying the raw ingress resources via kubectl instead of using the `stable/nginx-ingress` helm chart since we are using helm in most the other places?

Reply:
I tried it first :) but helm chart as is didn't work for kind. More athttps://github.com/kubernetes/ingress-nginx/issues/5569. Using the nginx-ingress chart will require a ton of workarounds applied on top of the chart.

Reply:
That likely explains the tons of special values I have around the `stable/nginx-ingress` in our kind scripts for my work environments 😄

Reply:
In the end values.yaml are expected to contain charts relevant to the example and not to provide a complete guide how to customize nginx-ingress :) .